transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionExtension.java

```diff
@@ -163,11 +169,12 @@ private void registerTransformers(SubsystemRegistration subsystem) {
 
 
         // 2.0.0 --> 1.5.0
-        ResourceTransformationDescriptionBuilder builderEap64 = chainedBuilder.createBuilder(subsystem.getSubsystemVersion(), MODEL_VERSION_EAP64);
+        ResourceTransformationDescriptionBuilder builderEap64 = chainedBuilder.createBuilder(MODEL_VERSION_WFLY10, MODEL_VERSION_EAP64);
         builderEap64.getAttributeBuilder()
                 .addRename(TransactionSubsystemRootResourceDefinition.USE_JOURNAL_STORE, CommonAttributes.USE_HORNETQ_STORE)
                 .addRename(TransactionSubsystemRootResourceDefinition.JOURNAL_STORE_ENABLE_ASYNC_IO, CommonAttributes.HORNETQ_STORE_ENABLE_ASYNC_IO)
-                .addRename(TransactionSubsystemRootResourceDefinition.STATISTICS_ENABLED, CommonAttributes.ENABLE_STATISTICS);
+                .addRename(TransactionSubsystemRootResourceDefinition.STATISTICS_ENABLED, CommonAttributes.ENABLE_STATISTICS)
+                .setDiscard(DiscardAttributeChecker.ALWAYS, TransactionSubsystemRootResourceDefinition.USE_ACTIONSTATUSSERVICE_RECOVERY_FILTER);
```

Comment:
Is there an example of this I can use as a reference please?

Is this a good example:
https://github.com/wildfly/wildfly/blob/9de9217e0106da10c7d05228bcf0559c81b6042c/connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/ConnectionDefinitionResourceDefinition.java#L115
https://github.com/wildfly/wildfly/blob/9de9217e0106da10c7d05228bcf0559c81b6042c/connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/ConnectionDefinitionResourceDefinition.java#L119